routes/journey/v0.js

```diff
@@ -60,8 +63,64 @@ const createJourney = async (req, res) => {
   return res.status(201).json(journeyResult);
 };
 
+/**
+ * 여정 수정
+ * @param {Request} req
+ * @param {Response} res
+ * @returns {Promise<*>}
+ */
+const updateJourney = async (req, res) => {
+  const { user } = res.locals.auth;
+  const { journeyIdx } = req.params;
+  let { title } = req.body;
+  const { value1, value2 } = req.body;
+
+  if (!title) throw new HttpBadRequest(Errors.JOURNEY.TITLE_MISSING);
+  if (!value1) throw new HttpBadRequest(Errors.JOURNEY.VALUES_MISSING);
+
+  [value1, value2].forEach((value) => {
+    if (value && !Object.values(Journey.VALUES).includes(value)) {
+      throw new HttpBadRequest(Errors.JOURNEY.VALUES_INCORRECT);
+    }
+  });
+
+  title = title.trim();
+
+  let journey;
+  try {
+    journey = await Journey.findByPk(journeyIdx);
+  } catch (e) {
+    throw new HttpInternalServerError(Errors.SERVER.UNEXPECTED_ERROR, e);
+  }
+
+  if (!journey) throw new HttpNotFound(Errors.JOURNEY.NOT_FOUND);
+
+  if (journey.userIdx !== user.idx) {
+    throw new HttpForbidden(Errors.AUTH.NO_PERMISSION);
+  }
+
+  try {
+    Journey.update(
+      {
+        title,
+        value1,
+        value2,
+      },
+      { where: { idx: journeyIdx, userIdx: user.idx } },
+    );
+  } catch (e) {
+    throw new HttpInternalServerError(Errors.SERVER.UNEXPECTED_ERROR, e);
+  }
```

Comment:
```suggestion
if(title) journey.title = title;
if(value1) journey.value1 = value1;
if(value2) journey.value2 = value2;
let journeyData;
  try {
   journeyData = journey.save( );
  } catch (e) {
    throw new HttpInternalServerError(Errors.SERVER.UNEXPECTED_ERROR, e);
  }
```
위에 수정할 journey 찾았으니까 해당 데이터 수정해주고 update 사용해주면 됩니다. 직접 테이블에서 또 찾아서 수정하지 않아도 될 것같아요.